scripts/run-module-bundlers-smoketests.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+const shell = require("shelljs");
+
+let result = 0;
+
+console.log(`
+Test webextension-polyfill bundled with webpack
+===============================================`);
+
+shell.exec("webpack --mode production --entry ./test/fixtures/bundle-entrypoint.js --output /tmp/webpack-bundle.js");
```

Comment:
we should early exit if webpack (or browserify) commands fails to generate the bundle.

Let's give a try to shelljs' set (as mentioned in the previous comment), if that doesn't work, let's explicitly check the code of these shell.exec results.